packages/liquidator/src/liquidator.js

```diff
@@ -282,61 +293,15 @@ class Liquidator {
         // minimim position size, as well as minimum liquidation to extend withdraw
         empMinSponsorSize: this.empMinSponsorSize,
         currentBlockTime,
+        // should bot emit specialized logs about skipping liquidations due to WDF
+        shouldEmitWDFSkipLogs: blockWindowHasRequestedWithdraw,
```

Comment:
i think this will trigger info logs when there have been withdraws which are expired. is this a problem?

Reply:
No if the withdraw has expired, then `hasWithdrawRequestPending` will be false so the liquidator will use its full balance to liquidate